Essentials/src/config.yml

```diff
@@ -574,6 +574,11 @@ is-water-safe: false
 # You should only change this to false if you use Minecraft China.
 safe-usermap-names: true
 
+# Should essentials.ext be a separate oermission from essentials.ext.others?
+# If false, those with essentials.ext can also extinguish others with explicitly having essentials.ext.others.
```

Comment:
- `oermission` -> `permission`
- `with` -> `without`?

Reply:
`separate oermission from essentials.ext.others` -> `separate permission to essentials.ext.others`